man/systemd.exec.xml

```diff
@@ -271,6 +271,25 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RootExtensions=</varname></term>
```

Comment:
I wonder how RootExtensions= is actually the right name for this? they don't actually extend the root of the hierarchy, after all, but /usr/ + /opt/ only.

Moreover so far we always had two distinct options for images and trees. i.e. in nspawn --directory= and --image=. for services RootImage= and RootDirectory=. It probably makes sense to continue this logic.

Hence using that as blueprint have two options, maybe named ExtensionImage= and ExtensionDirectory= instead?

Reply:
Sure thing - but ExtensionImages (plural) since it's a multi-value option. Also I don't have ExtensionDirectories ready - is it ok if it's a follow-up?